GODLIKE.m

```diff
@@ -54,7 +54,9 @@
 %
 %   popsize     positive integer. Indicates the TOTAL population 
 %               size, that is, the number of individuals of all 
-%               populations combined. 

```

Comment:
Hi @rodyo,

I assume this one is about passing an array of popsize for each algorithm. This is useful for the parallel execution feature. I was running on a 64-core machine and wanted to make sure that each algorithm's population size is a multiple of 64. By default, GODLIKE would randomly pick population sizes for each algorithm totaling to popsize. But in that case, 64 cores would be used unevenly and some of them would have to wait for others to finish. With this feature, the user can specify how many to run in each algorithm. Another solution would be to have a new option to ask to divide population equally among the algorithms, but I found that this solution is more flexible and doesn't require adding a new option.